include/seqan3/search/algorithm/search.hpp

```diff
@@ -87,23 +87,11 @@ inline auto search(index_t const & index, queries_t && queries, config_t const &
     }
     else
     {
-        // TODO: overload pipe operator for empty config object
```

Comment:
Can you point out the difference between if else blocks that you removed? I don't see the diffetrence and the connection to piping an empty config object.